django/contrib/gis/db/backends/spatialite/features.py

```diff
@@ -1,6 +1,8 @@
 from django.contrib.gis.db.backends.base.features import BaseSpatialFeatures
-from django.db.backends.sqlite3.features import \
-    DatabaseFeatures as SQLiteDatabaseFeatures
+from django.contrib.gis.geos import geos_version
+from django.db.backends.sqlite3.features import (
+    DatabaseFeatures as SQLiteDatabaseFeatures,
```

Comment:
FWIW, I submitted several patches to isort and the author was very responsive. I just didn't care enough about this one to fix it.